[4.1.0] | Release notes 4.1.0 (#1493)
dotnet/SqlClient: release-notes/4.1/4.1.0.md

# Release Notes

> ## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022

Author: ```suggestion
## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
```

This update brings the below changes over the previous preview release:

### Added

> - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)

Author: ```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#introduce-attestation-protocol-none)
```

> - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)

Reviewer: The `Read more` link is broken cause of using the special character "`". I'd suggest using a simpler title.

> - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)

Author: ```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](##use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```

> ### Use Attestation Protocol None to skip enclave attestation for VBS enclaves

Reviewer: NIT: Please update the #1488 accordingly.
```suggestion
### Introduce Attestation Protocol None
```

> ### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.

Author: ```suggestion
### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```

A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  

> Connection string examples:

Reviewer 2: ```suggestion
Connection string example:
```

# Release Notes

## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022

This update brings the below changes over the previous preview release:

### Added

- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)

### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.

A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  

Connection string examples:

```cs
//Attestation protocol NONE with no URL
"Data Source = {server}; Initial Catalog = {db}; Column Encryption Setting = Enabled; Attestation Protocol = None;"

> //Attestation protocol NONE with URL

Reviewer 2: I would remove this second connection string, just to emphasize that None should not be used with a URL.
